client/app/models/project.js

```diff
@@ -65,6 +65,8 @@ export default class ProjectModel extends Model {
 
   @attr('string') dcpProjectname;
 
+  @attr('string') dcpProjectid;
```

Comment:
In terms of process, these should be in a separate commit, instead of hidden in a commit indicating something completely different

Reply:
> In terms of process, these should be in a separate commit, instead of hidden in a commit indicating something completely different

I'm sorry — what do you mean by this? It's already being shipped with the response, and used in a template. How does the commit "indicate something completely different"?

Reply:
Confused about why you think this is some kind of process violation — it's a change that's necessary for this commit to function at all.

Reply:
Oh sorry, the green over the dcpFemafloodzoneshadedx made me think multiple attributes were being added. 
In this case, why can't `project.id` or `project.dcpName`  be used, instead of dcpProjectid?

Reply:
> Oh sorry, the green over the dcpFemafloodzoneshadedx made me think multiple attributes were being added.
> In this case, why can't `project.id` or `project.dcpName` be used, instead of dcpProjectid?

Because this model was set up to identify projects by their dcpName values, not the regular CRM ids.

We can't use dcpName because you can't link out to CRM projects with the dcpName value. It requires the GUID.

Reply:
Ah okay, gotcha, thank you